### DISCUSSION TITLE
Disable Power button LED blinking on low battery

### DISCUSSION THREAD

**Discussion by xneuromancer** [BartoszCichecki/LenovoLegionToolkit#660] [Ideas]
When a Legion 2021 reaches 20% battery or lower, the LED on the laptop power button blinks indefinitely until it's charged or dies.  This is really annoying.

It'd be great if you could figure out a way to disable this.

This has been asked a number of times in the Lenovo forums, and as far as I can tell, there's no way to disable it.

**Comment by BartoszCichecki** [comment]
There isn't any way to disable it as far as I can tell, but if someone knows a way, let me know.
